internal/server/grants_test.go

```diff
@@ -280,23 +270,18 @@ func TestAPI_ListGrants(t *testing.T) {
 				assert.NilError(t, err)
 
 				expected := []api.Grant{
-					{
-						User:      idInGroup,
-						Privilege: "custom1",
-						Resource:  "res1",
-					},
 					{
 						User:      idOther,
-						Privilege: "custom2",
-						Resource:  "res1.ns1",
+						Privilege: "connector",
+						Resource:  "res1.ns2",
```

Comment:
Nit: this grant doesn't make sense. It's not possible to grant `connector` to a Kubernetes namespace